spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -418,6 +418,27 @@ public void addDataPartitionedByDateToPartitioned() {
         sql("SELECT id, name, dept, date FROM %s ORDER BY id", tableName));
   }
 
+  @Test
+  public void addDataPartitionedByDateHourToPartitioned() {
```

Comment:
I think the test name is not capturing the problem its solving, should be something like 'testPartitionType' to capture the problem.

Reply:
updated name